exercises/Exercise_04_Estimation.md

```diff
@@ -1,76 +1,42 @@
+
+Read this file into a dataset named `penguins` and then filter down to two datasets, `adelie` and `chinstrap` using the following code:
+
+`adelie = penguins[penguins['species'] == 'Adelie'].dropna()`  
+`chinstrap = penguins[penguins['species'] == 'Chinstrap'].dropna()`
+
+1. Find the average `flipper_length_mm` by species (using the full dataset).
+2. Build a 95% confidence interval for the mean `flipper_length_mm` for adelie penguins.
+3. Build a 95% confidence interval for the mean `flipper_length_mm` for chinstrap penguins.
+4. Based on the confidence intervals you build, do you feel like it is safe to conclude that the population of Chinstrap penguins has a higher average flipper length than the population average flipper length for Adelie penguins?
+5. Find the average `body_mass_g` by species (using the full dataset).
+6. Build a 95% confidence interval for the mean `body_mass_g` for adelie penguins.
+7. Build a 95% confidence interval for the mean `body_mass_g` for chinstrap penguins.
+8. Based on the confidence intervals you build, do you feel like it is safe to conclude that the population of Chinstrap penguins has a higher average body mass than the population average body mass for Adelie penguins?
+
+### Part 2: Conceptual
+You are planning to build a confidence interval for the mean of some numeric variable. You have not yet gathered your sample, but based on past history, you expect that your sample will have a standard deviation around 10.
+
+1. If you are going to build a 95% confidence interval and want to end up with a margin of error less than 10, approximately what sized sample do you need?
+2. If you are going to build a 95% confidence interval and want to end up with a margin of error less than 1, approximately what sized sample do you need?
+3. If you are going to build a 95% confidence interval and want to end up with a margin of error less than 0.1, approximately what sized sample do you need?
+4. What do you notice about the relationship between the desired margin of error and the needed sample size?
+5. How do your answers change if you expect the standard deviation to be around 5 instead of 10?
+
+### Part 3: Squirrels
+Read the squirrel census data (2018_Central_Park_Squirrel_Census_-_Squirrel_Data.csv) into a DataFrame named `squirrels`.  
+
+1. For what proportion of encounters was it the case that the squirrel was seen approaching (as indicated in the `Approaches` column)?
+2. Build a 95% confidence interval for the proportion of squirrel encounters where the squirrel approaches. What is the margin of error of this confidence interval?
+3. Build a 95% confidence interval for the proportion of squirrel encounters where the squirrel is seen running (as indicated in the `Running` column). What is the margin of error of this confidence interval? How does the margin of error in this case differ from the margin of error the previous question? Why does this happen?
```

Comment:
slight nitpick here, missing an `in` in the second to last sentence: How does the margin of error in this case differ from the margin of error _in_ the previous question